content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -424,6 +424,39 @@ create ResourceClaim or ResourceClaimTemplate objects in namespaces labeled with
 `adminAccess` field. This ensures that non-admin users cannot misuse the
 feature.
 
+### Extended resource {#extended-resource}
```

Comment:
Rather than "Extended resource" as the name, I would use "Extended resource allocation by DRA". Here, as well as below.

Reply:
sure, changed as suggested